pinot-core/src/main/java/org/apache/pinot/core/segment/processing/framework/SegmentProcessorFramework.java

```diff
@@ -290,8 +290,7 @@ private List<File> generateSegment(Map<String, GenericRowFileManager> partitionT
           GenericRowFileRecordReader recordReaderForRange = recordReader.getRecordReaderForRange(startRowId, endRowId);
           SegmentIndexCreationDriverImpl driver = new SegmentIndexCreationDriverImpl();
           driver.init(generatorConfig, new RecordReaderSegmentCreationDataSource(recordReaderForRange),
-              RecordEnricherPipeline.getPassThroughPipeline(),
-              TransformPipeline.getPassThroughPipeline());
+              RecordEnricherPipeline.getPassThroughPipeline(), TransformPipeline.getPassThroughPipeline());
```

Comment:
nit: unnecessary formatting

Reply:
Yes did not mean to change this code . It reformatted when I did a "reformat code". Removed this formatting change.